### ISSUE TITLE
End User Apps: EUDA UI

### ISSUE THREAD

**Issue by rovats** [khartec/waltz#4669]
There is a requirement to display EUDA details in Waltz.

* Basic page to start with
  * Summary
  * Involvement
  * Entity Notes
* Hyperlink to EUDA pages from app lists (org unit page)
* Add to overall search

**Comment by davidwatkins73** [comment]
Probably better to simply make them applications in the first place (with a kind of `EUC`)

You may also be interested in the new 'promote' feature which takes a end user app (via the table) and turns it into a full app.

![image](https://user-images.githubusercontent.com/576829/74156781-f0b70200-4c0e-11ea-83b7-fbbc50d53a82.png)

**Comment by rovats** [comment]
Yes, making them full applications can be an option, but it will just clutter the `application` table with thousands of EUC records.
And we will need to apply filters on lots of dataloading jobs/custom views etc.

The promote feature will be useful to promote key EUDAs as full apps, but I think storing other EUDAs separately and creating a UI will be useful.

**Comment by MarkGuerriero** [comment]
> There is a requirement to display EUDA details in Waltz.
> 
> * Basic page to start with
>   
>   * Summary
>   * Involvement
>   * Entity Notes
> * Hyperlink to EUDA pages from app lists (org unit page)
> * Add to overall search

Hi Rohit  - is this need being driven by the new European regs?

**Comment by strangetobefree** [comment]
Indirectly yes, we are always looking for better ways to improve data lineage but there isn't a direct link to fulfilling a regulation requirement.

I think that EUDAs should be promotable but they should have their own screen and should be linkable to data flows as EUDAs as well. 

I'm worried that all EUDAs being applications would mean we would have to introduce filters wherever applications are counted. so it might be easier to have them as a separate entity

**Comment by davidwatkins73** [comment]
EUDAs have been lifted to a first-class to Waltz Entity in 1.60.
